src/app/templates/app/archive_requests.html

```diff
@@ -31,7 +31,7 @@
             <td>{{elem.submissionDatetime}}</td>
             <td>
               {% csrf_token %}
-              <input id='unarchive_button{{elem.id}}' class='btn btn-success' width='25px' value='Unarchive' />
+              <input type='button' id='unarchive_button{{elem.id}}' class='btn btn-success' width='25px' value='Unarchive' />
```

Comment:
We can change it to `<button>` tag instead `<input>`

Reply:
Replaced with `<button>` tag